_data/companies/wabbit.yml

```diff
@@ -0,0 +1,6 @@
+name: wabbit.pl
+wfh: Evangelist
```

Comment:
Could you explain what Evangelist means here?  Did you mean encouraged?

Reply:
No, we are a group of people working remotely for more than 10 years now for enterprise companies. We have created a service wabbit.pl for online meetings encouraging people to work and meet remotely. We believe our personal experience of remote work is unmatched to most people's imagination of remote work. We want to lead them through these hard times, give them a service in which they can meet family, run cultural meetings etc. We believe in Online Egagement and remote work is just a part of it.

Reply:
Ok, that makes a lot of sense. How about `100% Remote` in parentheses?

```suggestion
wfh: Evangelist (100% remote)
```

Reply:
@drewbrew, yes it looks ok, I don't seem to have an option to commit that change. Can you do it or should I reopen pull request?

Reply:
Ok, I'll take care of it next time I'm at the computer. Thanks!

Reply:
It turns out I don't have edit permissions. Do you see a "commit suggestion" button at the bottom of my suggestion?

Reply:
![image](https://user-images.githubusercontent.com/25050339/77301981-dfbedd80-6cf0-11ea-8886-1ec964ad9f1e.png)

No, I can't commit that suggestion. Please see attached

Reply:
Looks like GitHub has lost track of your source repository. I'll take your commit and open up a new PR with my fix on top of it.